tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../src")))
```

Comment:
Does this make my [Setup](https://tezos-reward-distributor-organization.github.io/tezos-reward-distributor/contributors.html) section obsolete?

Reply:
I resolved it